english/dance/bake-a-cake/bake-a-cake.md

```diff
@@ -9,3 +9,5 @@ Do you:
 [Sleep?](../../sleep/marshmallow.md)
 
 [Sing a song?](../../sing-song/sing.md)
+
+[Drink some coffee?](../../coffe/coffee.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.